ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -129,6 +151,19 @@
                     v-model="status"
                     clearable
                   />
+                  <v-select
+                    v-else-if="header.value === 'statusType'"
+                    :items="mhStatusTypes"
+                    hide-details
+                    single-line
+                    filled
+                    dense
+                    item-class="list-item"
+                    label="Status"
+                    :menu-props="{ bottom: true, offsetY: true }"
+                    v-model="status"
+                    clearable
+                  />
```

Comment:
Maybe i'm wrong but these look identical outside the `:items` value?
Would be it cleaner to apply the status types dynamically? ie ppr = statusTypes and mhr = mhrStatusTypes?

Reply:
The field names are the same, but the dropdown has different values based on Tab. Same applies for Registration types.  I found that if the use has access to both tabs that the dropdowns conflict.

Reply:
ok